smqtk_image_io/impls/gdal_io/__init__.py

```diff
@@ -0,0 +1 @@
+from .gdal_io import GdalImageReader
```

Comment:
Does this look like a clean implementation of the `impls` dir?

Reply:
A similar structure to the other breakouts would be
```
smqtk_image_io/
├── interfaces/
│   └── image_reader.py
└── impls/
    └── image_reader/
        ├── pil.py
        └── gdal.py
```

Reply:
corrected in latest commit